src/gui/rss/rsswidget.h

```diff
@@ -24,9 +26,8 @@
  * modify file(s), you may extend this exception to your version of the file(s),
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
- *
- * Contact : chris@qbittorrent.org arnaud@qbittorrent.org
  */
+
 #ifndef __RSS_IMP_H__
 #define __RSS_IMP_H__
```

Comment:
You forgot to change these names along the filename change.
Also i noticed you use `#pragma once` in half the header files but include guards in the other half.